src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -67,6 +67,24 @@
       <Link>snappy64.dll</Link>
     </Content>
   </ItemGroup>
+
+  <PropertyGroup>
+    <ZstdBinaries>Core/Compression/Zstd/lib/win</ZstdBinaries>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="$(ZstdBinaries)/libzstd.dll">
+      <Pack>true</Pack>
+      <PackagePath>x64/native/windows</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="$(ZstdBinaries)/libzstd.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Link>libzstd.dll</Link>
+    </Content>
+  </ItemGroup>
```

Comment:
I don't understand these packaging issues.

I defer to you and @vincentkam on them.